### ISSUE TITLE
Suggest running `updatedb` before using `locate`

### ISSUE THREAD

**Issue by Quba1** [hboetes/wiki#2]
As in title, I would like to suggest to add a note in cleanup section that users can run `updatedb` before running `locate`. Depending on the setup locate database may have old information and `locate` will show non-existing files leading to confusion.

**Comment by hboetes** [comment]
Good idea: https://github.com/hboetes/wiki/wiki/systemd%E2%80%90boot-on-fedora-40/_compare/023ec4a3f952eaeaf87dba7d2909904aeb835273...33e9f57bf78b4d7d9339c42ccf5f4c5427b60189
